v2.1/configure-replication-zones.md

```diff
@@ -758,36 +415,50 @@ There's no need to make zone configuration changes; by default, the cluster is c
     $ cockroach init --insecure --host=<any node hostname>
     ~~~
 
-2. On any node, configure a replication zone for the table that must be replicated more strictly:
+2. On any node, open the [built-in SQL client](use-the-built-in-sql-client.html):
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    # Create a YAML file with the replica count set to 5
-    # and the ssd attribute as a required constraint:
-    $ cat table_zone.yaml
+    $ cockroach sql --insecure
     ~~~
 
+3. Create a database and table:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > CREATE DATABASE db;
     ~~~
-    num_replicas: 5
-    constraints: [+ssd]
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > CREATE TABLE db.important_table;
     ~~~
 
+4. Configure a replication zone for the table that must be replicated more strictly:
+
     {% include copy-clipboard.html %}
-    ~~~ shell
-    # Apply the replication zone to the table:
-    $ cockroach zone set db.important_table \
-    --insecure \
-    --host=<any node hostname> \
-    -f table_zone.yaml
+    ~~~ sql
+    > ALTER TABLE CONFIGURE ZONE USING num_replicas = 5, constraints = '[+ssd]'
```

Comment:
Possible typo - Do you mean `ALTER TABLE db.important_table CONFIGURE ZONE ...` ?